crates/iota-indexer/tests/rpc-tests/indexer_api.rs

```diff
@@ -187,17 +188,15 @@ fn query_events_supported_events() {
 
 #[test]
 fn query_validator_epoch_info_event() {
-    let ApiTestSetup {
-        runtime,
-        store,
-        client,
-        cluster,
-    } = ApiTestSetup::get_or_init();
+    let ApiTestSetup { runtime, .. } = ApiTestSetup::get_or_init();
```

Comment:
If we only need the runtime, can't we just make it a `#[tokio::test]` test?

Reply:
This warrants refactoring all other similar tests as well. We benefit the running time.

Could you create an issue @tomxey ?

Reply:
Created https://github.com/iotaledger/iota/issues/4939

Reply:
The performance benefit may not be that big, since `get_or_init` just returns a shared cluster. But the code would be simpler.